query/group.sp

```diff
@@ -0,0 +1,17 @@
+query "azuread_group_not_public" {
+  sql = <<-EOQ
+    select
+      id as resource,
+      case
+        when visibility = 'Public' then 'alarm'
+        else 'ok'
+      end status,
+      case
+        when visibility = 'Public' then title || ' is public.'
+        else title || ' is not public.'
+      end reason
+      ${local.common_dimensions_sql}
+    from
+      azuread_group;
```

Comment:
@Priyanka-Chatterjee-2000  `azuread_group` table support tags any reason behind not adding tag dimenstion?